src/camera/camera.jl

```diff
@@ -12,8 +12,17 @@ end
 
 function disconnect!(c::Camera)
     for node in c.steering_nodes
-        filter!(listeners(node)) do x
-            !(x isa CameraLift) # remove all camera lifts
+        # remove all camera lifts
+        if node isa PriorityObservable
+            for f in listeners(node)
```

Comment:
Should we use the same code here for normal observables?

Reply:
Good point